# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bbkeys/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- bbkeys-0.9.0/src/Netclient.cpp.vanilla	2008-08-10 10:56:11.000000000 +0200
+++ bbkeys-0.9.0/src/Netclient.cpp	2008-08-10 10:56:18.000000000 +0200
@@ -26,6 +26,8 @@
 
 #include "Netclient.h"
 
+#include <string.h>
+
 Netclient::Netclient (const bt::Display &display)
   : bt::EWMH(display), _display(display)
 {
--- bbkeys-0.9.0/src/actions.cc.vanilla	2008-08-10 10:56:32.000000000 +0200
+++ bbkeys-0.9.0/src/actions.cc	2008-08-10 10:56:38.000000000 +0200
@@ -24,6 +24,7 @@
 
 #include "actions.hh"
 
+#include <stdlib.h>
 #include <iostream>
 #include <string>
 #include <sstream>
--- bbkeys-0.9.0/src/KeyClient.cpp.vanilla	2008-08-10 10:55:48.000000000 +0200
+++ bbkeys-0.9.0/src/KeyClient.cpp	2008-08-10 10:56:01.000000000 +0200
@@ -55,6 +55,7 @@
 #include "LocalUtil.h"
 #include "actions.hh"
 
+#include <string.h>
 #include <iostream>
 #include <algorithm>
 #include <vector>
--- bbkeys-0.9.0/src/Config.cpp.vanilla	2008-08-10 10:56:46.000000000 +0200
+++ bbkeys-0.9.0/src/Config.cpp	2008-08-10 10:56:56.000000000 +0200
@@ -24,9 +24,11 @@
 
 
 #include "Config.h"
-#include <string>
 
+#include <string.h>
+#include <string>
 #include <iostream>
+
 using std::cout;
 
 Config::Config() {
--- bbkeys-0.9.0/src/main.cpp.vanilla	2008-08-10 10:57:03.000000000 +0200
+++ bbkeys-0.9.0/src/main.cpp	2008-08-10 10:57:10.000000000 +0200
@@ -27,6 +27,8 @@
 
 #include "main.h"
 
+#include <string.h>
+
 //--------------------------------------------------------
 // parseOptions
 //--------------------------------------------------------
